migrations/etl/2_Mine_manager.sql

```diff
@@ -187,51 +175,38 @@ BEGIN
 
 
     RAISE NOTICE '.. Insert new MMS mine manager records into ETL_MANAGER';
-    --1. Mine with valid manager attached
-    WITH now_manager  AS(
-        SELECT
-            SUBSTRING(now_contact.cid from 1 for 7) AS mine_no      ,
-            contact_info.cid                        AS contact_cid  ,
-            contact_info.add_dt                     AS add_dt
-        FROM mms.mmsccc now_contact
-        INNER JOIN mms.mmsccn contact_info on
-            contact_info.cid=now_contact.cid_ccn
-        WHERE
-            SUBSTRING(now_contact.type_ind from 3 for 1) = 'Y'
+    --Step 1: Get regional mine_no's
+    WITH regional_mine AS (
+        SELECT mine_no
+        FROM mms.mmsmin
+        WHERE (min_lnk = 'N' OR min_lnk is null)
     ),
-    --2. Latest NoW with manager attached
-    latest_now AS(
-        SELECT
-            mine_no                 ,
-            MAX(add_dt)        add_dt
-        FROM now_manager
-        GROUP BY
-            mine_no
+
+    --Step 2, Get most recent NOW 
+    latest_now AS (
+        SELECT n.mine_no mine_no, Max(n.cid) cid
+        FROM mms.mmsnow n
+        WHERE n.mine_no in (SELECT mine_no from regional_mine)
+        GROUP BY n.mine_no
     ),
-    --3. Latest manager if more than 1 is attached
-    latest_manager AS (
-        SELECT
-            mine_no                         ,
-            MAX(contact_cid) AS contact_cid
-        FROM now_manager
-        WHERE mine_no||add_dt IN (
-            SELECT mine_no||add_dt
-            FROM latest_now
-        )
-        GROUP BY mine_no
+     --Step 3, get contact connection that is a Mine Manager
+    latest_now_ccc AS ( --contact connection
+        SELECT latest_now.mine_no mine_no, latest_now.cid cid, ccc.cid_ccn contact_cid
+        FROM mms.mmsccc ccc 
+        LEFT JOIN latest_now ON latest_now.cid = ccc.cid
+        WHERE SubStr(ccc.type_ind,3,1) = 'Y'
     ),
-    --4. Select new manager record
+    -- Step 4, get contact from contact connection
     new_manager AS (
-        SELECT
-            mine_no                         ,
-            contact_cid AS person_combo_id  ,
-            mine_no||contact_cid AS mgr_combo_id
-        FROM latest_manager
-        where mine_no||contact_cid NOT IN (
+        SELECT latest_now_ccc.mine_no mine_no, cn.cid person_combo_id, latest_now_ccc.mine_no||cn.cid as mgr_combo_id
```

Comment:
Could you put the selected columns on separate lines to make it easier to read with the aliases?

Reply:
done